packages/runtime-common/indexer.ts

```diff
@@ -503,6 +529,126 @@ export class Indexer {
     ];
   }
 
+  async searchPrerendered(
+    realmURL: URL,
+    { filter, sort, page }: Query,
+    loader: Loader,
+    opts: QueryOptions = {},
+  ): Promise<{
+    prerenderedCards: PrerenderedCard[];
+    prerenderedCardCssItems: PrerenderedCardCssItem[];
+    meta: QueryResultsMeta;
+  }> {
+    let { results, meta } = await this._search(
+      realmURL,
+      { filter, sort, page },
+      loader,
+      opts,
+      'SELECT url, ANY_VALUE(file_alias) as file_alias, ANY_VALUE(embedded_html) as embedded_html',
+    );
+
+    let cardFileAliases = results.map((c) => c.file_alias);
+    let realmVersion = meta.page.realmVersion;
+
+    let conditions: CardExpression[] = [
+      ['i.realm_version = ', param(realmVersion)],
+      ['i.type =', param('instance')],
+      [
+        `i.file_alias IN (${cardFileAliases
+          .map((fileAlias) => `'${fileAlias}'`)
```

Comment:
this would be better as `param(fileAlias)` and then _don't_ do a join on the string. rather just use a spread to add this to the outer `CardExpression`